build.sbt

```
@@ -41,7 +41,7 @@ resolvers ++= Seq(
 )
 
 libraryDependencies ++= Seq(
-  "org.scalatest" %% "scalatest" % "3.0.8",
+  "org.scalatest" %% "scalatest" % "3.2.0",
```

```suggestion
  "org.scalatest" %% "scalatest" % "3.0.8",
  "org.scalatest" %% "scalatest" % "3.2.0" % "test",
```

This should allow the ChiselTest infrastructure to be used with projects requiring 3.0.8 (or later), but having the ChiselTest tests run on ScalaTest 3.2.0. Projects with a dependency on ChiselTest will import ScalaTest 3.0.8 by default (I think? - at least compiles with 3.0.8-style code without deprecation warnings), but can use a later version of ScalaTest by adding a later versioned dependency.

---

FYI, here are the results of testing with a project written for ScalaTest 3.0.8 and depending on chiseltest, with chiseltest published locally with the above modification.

---

build.sbt contains:
```scala
libraryDependencies += "edu.berkeley.cs" %% "chisel3" % "3.4-SNAPSHOT"
libraryDependencies += "edu.berkeley.cs" %% "chiseltest" % "0.3-SNAPSHOT"
```
compiles fine, with no ScalaTest related warnings

---

build.sbt contains:
```scala
libraryDependencies += "edu.berkeley.cs" %% "chisel3" % "3.4-SNAPSHOT"
libraryDependencies += "org.scalatest" %% "scalatest" % "3.1.0" % "test"
libraryDependencies += "edu.berkeley.cs" %% "chiseltest" % "0.3-SNAPSHOT"
```
produces deprecated warnings from ScalaTest

---

build.sbt contains:
```scala
libraryDependencies += "edu.berkeley.cs" %% "chisel3" % "3.4-SNAPSHOT"
libraryDependencies += "org.scalatest" %% "scalatest" % "3.2.0" % "test"
libraryDependencies += "edu.berkeley.cs" %% "chiseltest" % "0.3-SNAPSHOT"
```
produces ScalaTest name related compile time errors